partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
Yes good spot - for some reason the class `panel-indentvalidation-error` actually colours the border red, perhaps the css selector is testing the end of the class?  

Default behaviour in hof forms is to colour the panel border red, perhaps it would be worth adding a flag to the field config for this? Thoughts @JoeChapman @gavboulton @easternbloc?

![](https://gyazo.com/60623ded57a566a6bbed8c9ca31fd279)

vs

![](https://gyazo.com/e159b1c9b8a8b82017e0b7f77deb34cf)